src/main/host/thread_preload.c

```diff
@@ -21,6 +22,19 @@
 
 #define THREADPRELOAD_TYPE_ID 13357
 
+// We use an int here because the option parsing library doesn't provide a way
+// to set a boolean flag to false explicitly.
+static gint _sendExplicitBlockMessage = true;
+OPTION_EXPERIMENTAL_ENTRY(
+    "send-explicit-block-message", 0, 0, G_OPTION_ARG_INT, &_sendExplicitBlockMessage,
+    "Send message to plugin telling it to stop spinning when a syscall blocks", "[0|1]")
+
+static gint _spinMax = 8096;
+OPTION_EXPERIMENTAL_ENTRY(
+    "preload-spin-max", 0, 0, G_OPTION_ARG_INT, &_spinMax,
+    "Max number of iterations to busy-wait on ICP sempahore before blocking. -1 for unlimited.",
+    "[n]")
```

Comment:
I think with our current set of options, we specify the default value of the option in brackets at the end of the description string (line 35), and I don't think we use brackets for the part in line 36. We should make our use of brackets consistent, and also I think we want to specify the default value.

Reply:
Done